vmdb/lib/migration_helper.rb

```diff
@@ -40,33 +40,11 @@ def change_id_columns(table, id_cols, type)
   #
 
   def add_pk(*args)
-    meth = "add_pk_#{connection.adapter_name.downcase}"
-    self.send(meth, *args) if self.respond_to?(meth)
+    # PUNT
```

Comment:
Yeah, I'm thinking dropping it as well.  It was used in the [change_id_column](https://github.com/ManageIQ/manageiq/blob/28b598ed5485a31a8d617b45c00856d23bb7dbcc/vmdb/lib/migration_helper.rb#L28-32) method to drop and readd the primary key where necessary when changing an `id` column's type, but this was only necessary for SQL server.  Since SQL server is dropped, we can probably just throw that method away.  In fact it looks like the `change_id_column` can probably be thrown away as well.